src/tests/ftest/daos_test/shm.yaml

```diff
@@ -0,0 +1,20 @@
+hosts:
+  test_servers: 1
+timeout: 90
+server_config:
+  name: daos_server
+  engines_per_host: 1
+  engines:
+    0:
+      targets: 4
+      nr_xs_helpers: 0
+      storage:
+        0:
+          class: ram
+          scm_mount: /mnt/daos
+  system_ram_reserved: 1
+pool:
+  scm_size: 1G
+container:
+  type: POSIX
+  control_method: daos
```

Comment:
This isn't used by the test.  A typical cmocka test would use the `pool` entry information, but only when the test is run via `DaosCoreBase.run_subtest()`:
- https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/daos_core_base.py#L68-L69
- https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/daos_core_base.py#L89-L90

Reply:
The new test "shm_test" does not need scm_size and nvme_size information.

Reply:
I think what @phender means is since this test does not use `DaosCoreBase.run_subtest()`, the entire `pool` and `container` keys here are not used and can be removed

Reply:
... or if we use `DaosCoreBase.run_subtest()` we can keep it

Reply:
@daltonbohning @phender Thank you very much! I will try removing pool and container keys locally to make sure it works. I thought they are required.

Reply:
@daltonbohning @phender You are right. We can remove pool and container keys in yaml file as you suggested. I will update it in next commit. Thank you!